framework/auth/oauth_scopes.py

```diff
@@ -111,7 +113,7 @@ class ComposedScopes(object):
     NODE_ALL_WRITE = NODE_ALL_READ + NODE_METADATA_WRITE + NODE_DATA_WRITE + NODE_ACCESS_WRITE
 
     # Full permissions: all routes intended to be exposed to third party API users
-    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ
+    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + (CoreScopes.GUIDS_READ, )
     FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE
```

Comment:
Since you didn't make a GUIDS composed scope, and since it looks like GUIDS_READ isn't in the above scopes, I'm pretty sure you'll need to add your GUIDS_READ scope here as well.